### PR TITLE
das sql script

### PR DESCRIPTION
**Pull request by ttttux**
Joa, hier ist der sql part shrubi, falls es nt passt schreib mir

### CODE REVIEW

File: db/lehriumdb.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+);
+
+CREATE TABLE IF NOT EXISTS User (
+    id SERIAL PRIMARY KEY,
+    email CHAR(255) UNIQUE,
+    password CHAR(255),
+    roles VARCHAR(50)[],
+    untisUsername CHAR(100) UNIQUE,
+    teacherIds VARCHAR(10)[]
+);
```
Comment (ChikyuKido) [inline db/lehriumdb.sql:26]: teacherIds and roles should be a char

Comment (ttttux) [reply db/lehriumdb.sql:26]: sry, I forgot

File: db/lehriumdb.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+CREATE TABLE IF NOT EXISTS Teacher (
+    id SERIAL PRIMARY KEY,
+    name CHAR(50),
+    middleName CHAR(50),
+    lastName CHAR(50),
+    shortName CHAR(3) UNIQUE
+);
+
+CREATE TABLE IF NOT EXISTS User (
+    id SERIAL PRIMARY KEY,
```
Comment (ChikyuKido) [inline db/lehriumdb.sql:20]: change to users

File: db/lehriumdb.sql [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+-- gut mit postgres aus, also muesst ihr die db
+-- (nennt sie LehriumDB) manuel erstellen, das geht so:
+-- sudo su - postgres
+-- createdb LehriumDB
+-- grant all privileges on database LehriumDB to postgres;
+
+\c LehriumDB
+
+CREATE TABLE IF NOT EXISTS Teacher (
+    id SERIAL PRIMARY KEY,
```
Comment (ChikyuKido) [inline db/lehriumdb.sql:12]: change to Teachers

**Comment by ChikyuKido** [comment]
Sry. We don't need the script :(

**Pull request closed without merging**